app/presenters/attachment_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class AttachmentPresenter
+
+  def initialize(attachment)
+    @attachment = attachment
+  end
+
+  def to_json
+    {
+      url: @attachment.url,
+      title: @attachment.title,
+      content_type: @attachment.content_type,
+      updated_at: updated_at,
+      created_at: created_at,
+      content_id: @attachment.content_id
+    }
+  end
+
+  private
```

Comment:
The [style guide](https://github.com/alphagov/styleguides/blob/master/ruby.md#classes) reckons `private` should be on the same indent level as class. I think the linter usually catches this. Is it enabled on this repo?